.goreleaser.yml

```diff
@@ -0,0 +1,32 @@
+dist: release
+env:
+  - GO111MODULE=on
+before:
+  hooks:
+    - go mod download
+    - go mod tidy
```

Comment:
why are these lines needed?

Reply:
They recommended that in the instructions. I don't think they're needed/required.